marconi-chain-index/test-lib/Gen/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -29,26 +29,29 @@ import Marconi.ChainIndex.Indexers.Utxo qualified as Utxo
 --
 --   * that any generated UTXO is unique
 --   * for any spent tx output, there must be a UTXO created in a previous event
-genUtxoEvents :: Gen [StorableEvent UtxoHandle]
+genUtxoEvents :: Gen [(StorableEvent UtxoHandle, [C.Tx C.AlonzoEra])]
```

Comment:
The comment 'Generates a list of UTXO events.' should change.

I'd like to propose a name change for this function as it is doing more than generating UtxoEvents. Maybe `genUtxoEventsAndTX` ?
And then change the current genUtxoEvents as such:
```
genUtxoEvents :: Gen [StorableEvent UtxoHandle]
genUtxoEvents = genUtxoEventsAndTX <&> fmap fst
```

Reply:
Thanks :) Done